### ISSUE TITLE
`npm test` fails on Windows 10

### ISSUE THREAD

**Issue by skfd** [hodgepodgers/ng-intl-tel-input#36]
It must be some typical problem, but not sure how to fix this. Here's the output:

```
> ng-intl-tel-input@1.2.1 jshint C:\Users\ksenk_000\Documents\GitHub\ng-intl-tel-input
> ./node_modules/jshint/bin/jshint *.js

'.' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Users\\ksenk_000\\AppData\\Roaming\\npm\\node_modules\\np
m\\bin\\npm-cli.js" "run" "jshint"
npm ERR! node v4.2.1
npm ERR! npm  v3.3.9
npm ERR! code ELIFECYCLE
npm ERR! ng-intl-tel-input@1.2.1 jshint: `./node_modules/jshint/bin/jshint *.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ng-intl-tel-input@1.2.1 jshint script './node_modules/jshint/bin/jshint *.js'.
npm ERR! This is most likely a problem with the ng-intl-tel-input package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./node_modules/jshint/bin/jshint *.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls ng-intl-tel-input
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\ksenk_000\Documents\GitHub\ng-intl-tel-input\npm-debug.log
npm ERR! Test failed.  See above for more details.
C:\Users\ksenk_000\Documents\GitHub\ng-intl-tel-input [master]> npm karma

Usage: npm <command>

where <command> is one of:
    access, add-user, adduser, apihelp, author, bin, bugs, c,
    cache, completion, config, ddp, dedupe, deprecate, dist-tag,
    dist-tags, docs, edit, explore, faq, find, find-dupes, get,
    help, help-search, home, i, info, init, install, issues, la,
    link, list, ll, ln, login, logout, ls, outdated, owner,
    pack, ping, prefix, prune, publish, r, rb, rebuild, remove,
    repo, restart, rm, root, run-script, s, se, search, set,
    show, shrinkwrap, star, stars, start, stop, t, tag, team,
    test, tst, un, uninstall, unlink, unpublish, unstar, up,
    update, upgrade, v, verison, version, view, whoami

npm <cmd> -h     quick help on <cmd>
npm -l           display full usage info
npm faq          commonly asked questions
npm help <term>  search for help on <term>
npm help npm     involved overview

Specify configs in the ini-formatted file:
    C:\Users\ksenk_000\.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@3.3.9 C:\Users\ksenk_000\AppData\Roaming\npm\node_modules\npm
```

**Comment by bighappyface** [comment]
@skfd please pull down v1.3.0 and let me know if it solves the run on Windows.

**Comment by skfd** [comment]
Some minor issue, but I managed to fix: #38 .

I made a patch that allows binding current country. That's for a common scenario when there is a country control.

Should I add test and pull request? It's basically this:

```js
        scope: {
          country: '='
        },
            //....
            scope.$watch('country', function(newValue) {
            if (newValue !== null && newValue !== void 0 && newValue !== '') {
              elm.intlTelInput("selectCountry", newValue);
            }
```

**Comment by bighappyface** [comment]
Closed with https://github.com/hodgepodgers/ng-intl-tel-input/pull/38
